spreadsheet/formula/binaryexpr.go

```diff
@@ -403,3 +403,31 @@ func listValueOp(op BinOpType, lhs []Result, rhs Result) Result {
 	}
 	return MakeListResult(res)
 }
+
+// Eval evaluates the binary expression using the context given.
+func (b BinaryExpr) String() string {
+	opStr := ""
+	switch b.op {
+		case BinOpTypePlus: opStr = "+"
+		case BinOpTypeMinus: opStr = "-"
```

Comment:
Should be using goimports?  I expect that the opStr = ... would go onto the following line